CHANGES

```diff
@@ -3,6 +3,24 @@ repositories (changes that are automatically handled by the format upgrade tools
 are not marked). Those prefixed with "(+)" are new command/option (since
 2.1.0~alpha2).
 
+2.2.1:
+* (*) Fix a regression in `opam install --deps-only` where the direct
+  dependencies were not set as root packages [#6125 @rjbou]
+* (*) Fix a regression when building git packages where the build result
+  were sometimes inconsistent compared to outside opam, by disabling shallow
+  clone by default except when fetching an opam repositories
```

Comment:
```suggestion
* (*) Fix a regression on git fetching by disabling shallow
  clone by default except when fetching an opam repositories
```
It is not only git package build that was affected

Reply:
Do you have an example that isn't a git package?

Reply:
oh you mean local (rsync), sure. I can replace by "packages using git"

Reply:
mh, actually no the issue wouldn't be triggered there. Do you have an example?

Reply:
The entry doesn't say it's failing. Only that the result is inconsistent between with and without opam. That does englobe all that in my opinion

Reply:
fixed